### PR TITLE
Version 4.6.1 -> 4.7.0

### PR DESCRIPTION
**Pull request by olexandr-konovalov**
I've incremented version numbers, and also converted CHANGES and README to markdown so they are now better rendered on GitHub and also on package homepage. If looks ok to @Stefan-Kohl I will be able to publish the release.

### CODE REVIEW

**Comment by olexandr-konovalov** [comment]
@Stefan-Kohl any possibility of reviewing shortly for a chance of catching GAP 4.9.2 release?

**Comment by Stefan-Kohl** [comment]
@alex-konovalov  Thank you! -- Though as all changes since RCWA 4.6.0 of December 5, 2017 were of a purely janitorial nature, I'd rather increment the patch number -- i.e. the next release would be 4.6.2 rather than 4.7.0.  -- At least so far I have incremented the second number only if there were changes in functionality or documentation since the previous release -- whereas for bug fixes and the like, rather the patch level is incremented. -- This would also be consistent with FactInt, where the move to GitHub also just led to an increment of the patch level (1.6.0->1.6.1). Thanks again.

In short -- if you change 4.7.0 to 4.6.2, I will "rebase and merge", and you are welcome to go ahead with making the release.

**Comment by olexandr-konovalov** [comment]
@Stefan-Kohl ok, thanks! Please watch #5 then, and merge it as soon as CI tests will pass.

**Comment by Stefan-Kohl** [comment]
@alex-konovalov Done. -- Thank you!
